docs/source/getting-started/installation.rst

```diff
@@ -92,7 +95,7 @@ your installation of Python 3: on most systems, this is ``python3``, but yours m
 
    $ make install
 
-If you think something went wrong, check the log files under ``logs/install/``. If you want to try again, you can use ``make clean`` to delete any build files before running ``make install`` again.
+If you think something went wrong, check the log files in ``logs/``. If you want to try again, you can use ``make clean`` to delete any build files before running ``make install`` again.
```

Comment:
The log restructure is a great solution for now. It keeps the "finding what went wrong" process simple